### PR TITLE
Fix netskope schema mismatch

### PR DESCRIPTION
**Pull request by evisochek**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-8118

## Description
Refactor the compatibility between the data model rules and the schema hints

### CODE REVIEW

**Comment by eepstain** [comment]
Due to a new validation-
https://jira-hq.paloaltonetworks.local/browse/CIAC-8101
All of the packs without TestData configs that require an update should be force merged.

**Review by eepstain — APPROVED** [review]

File: Packs/Netskope/ModelingRules/NetskopeEventCollector_1_3/NetskopeEventCollector_1_3_schema.json [hunk 1]
```diff
@@ -235,6 +232,128 @@
...
+      "dst_country": {
+        "type": "string",
+        "is_array": false
+    },
+      "dst_latitude": {
+        "type": "string",
+        "is_array": false
+        
+    },
+        "dst_longitude": {
```
Comment (tkatzir) [inline Packs/Netskope/ModelingRules/NetskopeEventCollector_1_3/NetskopeEventCollector_1_3_schema.json:307]: Indentation here is different from the indentation above?

File: Packs/Netskope/ModelingRules/NetskopeEventCollector_1_3/NetskopeEventCollector_1_3_schema.json [hunk 2]
```diff
@@ -235,6 +232,128 @@
...
+        "dst_timezone": {
+          "type": "string",
+          "is_array": false
+
+    },
+        "server_bytes": {
+          "type": "int",
+          "is_array": false
+
+
```
Comment (tkatzir) [inline Packs/Netskope/ModelingRules/NetskopeEventCollector_1_3/NetskopeEventCollector_1_3_schema.json:326]: Why those blank lines?

File: Packs/Netskope/ModelingRules/NetskopeEventCollector_1_3/NetskopeEventCollector_1_3_schema.json [hunk 3]
```diff
@@ -235,6 +232,128 @@
...
+        
+    },
+      "src_longitude": {
+        "type": "string",
+        "is_array": false
+    },
+      "src_timezone": {
+        "type": "string",
+        "is_array": false
+
```
Comment (tkatzir) [inline Packs/Netskope/ModelingRules/NetskopeEventCollector_1_3/NetskopeEventCollector_1_3_schema.json:286]: Is this blank line needed?
See below for many more.

**Review by tkatzir — COMMENTED** [review]
See comments.

**Pull request merged**